tokio/src/runtime/thread_pool/queue/global.rs

```diff
@@ -51,7 +51,7 @@ impl<T: 'static> Queue<T> {
     /// Close the worker queue
     pub(super) fn close(&self) -> bool {
         // Acquire the lock
-        let _p = self.pointers.lock().unwrap();
+        let p = self.pointers.lock().unwrap();
```

Comment:
Are the explicit drops part of the fix, or are they left over from debugging?

Reply:
They are not part of the fix, but given that the mutex protects more than just the  data, I thought it would be good to be explicit.